lib/test.js

```diff
@@ -347,9 +347,11 @@ Test.prototype.pragma = function (set) {
     return
   }
 
-  for (var i in set) {
-    this.push('pragma ' + (set[i] ? '+' : '-') + i + '\n')
-  }
+  Object.keys(set).forEach(function (i) {
+    if (set.hasOwnProperty(i)) {
+      this.push('pragma ' + (set[i] ? '+' : '-') + i + '\n')
+    }
+  })
```

Comment:
`this` will be different in the forEach method.  This line should be `  }, this)` to explicitly bind the forEach iterator.